.github/workflows/build-docker.yml

```diff
@@ -63,7 +83,8 @@ jobs:
           file: ./docker/centrifuge-chain/Dockerfile
           build-args: |
             FEATURES=${{ matrix.target == 'test' && 'fast-runtime' || '' }}
-          push: ${{ github.event_name != 'pull_request' }}
+          # Always push, either to ghcr or Dockerhub
+          push: true
```

Comment:
Question: Is this not exactly the opposite of the PR expected behavior?

Reply:
+1

Reply:
with this PR, PRs will always push to the github docker repo, but only if the dockerfile has been modified, this allows the developer to test the docker container in the real world before closing the PR on changes to the docker components/image.
```
pull_request:
    paths:
      - ".github/workflows/build-docker.yml"
      - "docker/centrifuge-chain"
      - "docker/scripts"
      - ".dockerignore"
 ```